ClassModules/Priest.lua

```diff
@@ -4572,6 +4572,10 @@ if classIndexId == 5 then --Only do this if we're on a Priest!
 						barBorderColor = specSettings.colors.bar.deathsTormentMax.color
 					end
 					
+					if specSettings.colors.bar.mindDevourer.enabled and TRB.Functions.Class:IsValidVariableForSpec("$mdTime") then
```

Comment:
Slight adjustment to use the snapshot variable instead of a bar text variable lookup -- I know these are used in a few places as a shortcut but I've been trying to remove them as much as possible just to keep calls to very large if/else block functions down.

```suggestion
					if specSettings.colors.bar.mindDevourer.enabled and snapshots[spells.mindDevourer.id].buff.isActive then
```